modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -508,6 +514,18 @@ AUI.add(
 				return instance._getFieldOptions(value).length > 0 && instance._getFieldType(value) !== 'text';
 			},
 
+			_isNumber: function(operandTypeValue) {
+				var instance = this;
+
+				return operandTypeValue === 'double' || operandTypeValue === 'integer';
```

Comment:
Here too,If you don't want to compare the type of the data, you just need to use two equals (==), because is more performative. But let me know your thinking.